<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCDGUI</class>
 <widget class="QMainWindow" name="BCDGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BCDGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="dec_edit1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>113</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dec_edit2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>113</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="bcd1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>481</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>bcd1</string>
    </property>
   </widget>
   <widget class="QLabel" name="bcd2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>481</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>bcd2</string>
    </property>
   </widget>
   <widget class="QLabel" name="plus_sign">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="dec_header">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Dec</string>
    </property>
   </widget>
   <widget class="QLabel" name="bcd_header">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>119</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>BCD</string>
    </property>
   </widget>
   <widget class="QLabel" name="sum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="dec_sum">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>119</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>dec sum</string>
    </property>
   </widget>
   <widget class="QLabel" name="bcd_sum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>491</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>bcd sum</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
